### ISSUE TITLE
autorandr-launcher not compiled with LDFLAGS

### ISSUE THREAD

**Issue by dvzrv** [phillipberndt/autorandr#195]
Hi! I'm packaging autorandr for Ach Linux. When building 1.10.0 I saw that there's now a new tool (autorandr-launcher) being built.

Unfortunately the executable is compiled in the `install_launcher` target and does not accept `LDFLAGS`: https://github.com/phillipberndt/autorandr/blob/76a229a5e7b71c1c1eb7151556621987dc231dc6/Makefile#L150

This leads to a) compiling while installing, which is bad practice and b) a file that lacks [full RELRO](https://ctf101.org/binary-exploitation/relocation-read-only/) because the exported LDFLAGS (`LDFLAGS="-Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now"`) during build are ignored.

Output from [namcap](https://wiki.archlinux.org/index.php/Namcap) (Arch Linux' package checking tool):

```
autorandr W: ELF file ('usr/bin/autorandr-launcher') lacks FULL RELRO, check LDFLAGS.
```

It would be better compile and then install in a separate target and additionally allow external `$LDFLAGS`

**Comment by phillipberndt** [comment]
Good points. I didn't keep that in mind when switching the defaults. Would you like to see a point release on top?

**Comment by dvzrv** [comment]
Whatever is most convenient for you.

I'm currently using

```
  gcc -Wall $CFLAGS contrib/autorandr_launcher/autorandr_launcher.c \
      -o contrib/autorandr_launcher/autorandr-launcher \
      $(pkg-config --libs --cflags pkg-config xcb xcb-randr) $LDFLAGS
```
and am not using the Makefile.
This way I can install (optimized) `.pyc` files for `autorandr.py` and install the file as a package (using setuptools).

**Comment by phillipberndt** [comment]
Alright. I just released 1.10.1, which reverts the change that makes the launcher the default.

Rules for compiling the launcher are fixed as well, but I currently believe that sticking to udev/systemd is the safer bet.
